modules/Sources/Features/OnboardingFlow/PlainOnboardingView.swift

```diff
@@ -23,38 +22,33 @@ public struct PlainOnboardingView: View {
     public var body: some View {
         WithViewStore(store) { viewStore in
             VStack {
-                VStack(alignment: .leading, spacing: 80) {
-                    Text(L10n.PlainOnboarding.title)
-                        .font(
-                            .custom(FontFamily.Inter.regular.name, size: 34, relativeTo: .largeTitle)
-                            .weight(.heavy)
-                        )
-
-                    Text(L10n.PlainOnboarding.caption)
-                        .font(
-                            .custom(FontFamily.Inter.regular.name, size: 16, relativeTo: .body)
-                            .weight(.semibold)
-                        )
-                }
-                .padding(0)
+                Asset.Assets.welcomeScreenLogo.image
+                    .resizable()
+                    .frame(width: 169, height: 160)
+                    .padding(.top, 60)
                 
+                Text(L10n.PlainOnboarding.title)
+                    .font(.custom(FontFamily.Inter.regular.name, size: 22))
+                    .padding(.top, 25)
+                    .multilineTextAlignment(.center)
+
                 Spacer()
                 
                 Button(L10n.PlainOnboarding.Button.createNewWallet.uppercased()) {
-                    viewStore.send(.createNewWallet, animation: .easeInOut(duration: animationDuration))
+                    viewStore.send(.createNewWallet)
                 }
                 .zcashStyle()
-                .padding(.horizontal, 70)
+                .frame(minWidth: 236)
                 .padding(.bottom, 30)
 
                 Button(L10n.PlainOnboarding.Button.restoreWallet.uppercased()) {
-                    viewStore.send(.importExistingWallet, animation: .easeInOut(duration: animationDuration))
+                    viewStore.send(.importExistingWallet)
                 }
                 .zcashStyle(.secondary)
-                .padding(.horizontal, 70)
-                .padding(.bottom, 20)
+                .frame(minWidth: 236)
+                .padding(.bottom, 50)
             }
-            .padding(.all)
+            .padding(.horizontal, 70)
```

Comment:
Just an idea, if these repeating padding sizes across the screen couldn't be extracted to some commonplace. On Android, we use Dimens for such repeatedly used constant measures. This question applies to several padding/sizes in these changes.

Reply:
I'm aware of this but up to this point I'm still not convinced that the rule applies to all screens or designs. Some screens in figma have 58, 60, 70, ... different paddings. The fact it's 70 several times is purely my decision. So I would rather wait till there is some alpha and feedback from Richard before I start to implement shared constraints.